ami/main/charts.py

```diff
@@ -269,7 +269,7 @@ def event_top_taxa(event_pk: int, top_n: int = 10):
         .values("name")
         # .annotate(num_detections=models.Count("occurrences__detections"))
         .annotate(num_detections=models.Count("occurrences"))
-        .order_by("-num_detections")[:top_n]
+        .order_by("num_detections")[:top_n]
```

Comment:
Isn't this lowest to highest?

Reply:
Yes, you're right. I just tested out the chart by modifying the species count and made the corrected changes. Here's an example top species chart:
<img width="393" alt="Screenshot 2024-09-18 at 9 51 26 AM" src="https://github.com/user-attachments/assets/8edc30da-4a87-4d13-b8ae-a0b145498035">

If top_n=1, then it looks like this:
<img width="394" alt="Screenshot 2024-09-18 at 9 53 02 AM" src="https://github.com/user-attachments/assets/64b0a6a6-f3ca-4185-b8bd-9bd2bbf83be7">

Note, this is what the test data looks like which modified [here](https://github.com/RolnickLab/ami-platform/blob/f4807a1c97a2f2b4e06e00febbcd1f9a084ad1b9/tests/fixtures/main.py#L157).
<img width="283" alt="Screenshot 2024-09-18 at 9 54 42 AM" src="https://github.com/user-attachments/assets/e8bfcf0d-d6a5-4f73-b280-0ba216a5a7b1">

@mihow Is the plot showing the correct data? I thought that the plot would show 2 occurrences of _Vanessa atalanta_ and 1 occurrence of _Vanessa itea_?

Reply:
Thanks Vanessa,

The `TEST_TAXA_CSV_DATA` is for Taxon entries, not Occurrence entries. So adding duplicates there won't have the effect you are looking for. 

The charts are showing which occurrences have been IDed to which Taxa. In the test data, you want to modify the number of occurrences that are generated for each taxon, rather than the taxonomy _choices_, if that makes sense?

Also if you make your self "is_staff" in the Django admin, then you can modify the occurrence IDs pretty quickly without regenerating. But I like both approaches (the test data should have different numbers of IDs for different species).

Reply:
I think I understand; where is the test data of the occurrences generated/modified? Based on my testing using the admin, the Top Species chart appears to work -- see my [other comment below](https://github.com/RolnickLab/antenna/pull/580#issuecomment-2555920987)